### PR TITLE
Bumped qross{core,python}-9999 to build with Qt5.

### PR DESCRIPTION
**Pull request by 0xd34df00d**

### CODE REVIEW

**Comment by 0xd34df00d** [comment]
@SoapGentoo ping.

File: dev-libs/qrosscore/qrosscore-9999-r1.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 EGIT_REPO_URI="git://github.com/0xd34df00d/Qross.git"
```
Comment (SoapGentoo) [inline dev-libs/qrosscore/qrosscore-9999-r1.ebuild]: it's generally nicer to have `EGIT_REPO_URI` after the `inherit git-r3` line

**Comment by a17r** [comment]
There's no reason to create 9999-r1 ebuilds, just convert the 9999 ebuilds in place.

**Comment by 0xd34df00d** [comment]
@a17r I couldn't think of a way to do this atomically enough to always have a sane PM tree state.

Also, this changes the files on disk, and if a dependency assumes qross is built with Qt5 (as it is now), while it was built long ago with Qt4 (without revbump), things will blow up.

**Comment by a17r** [comment]
Well, changes on disk happen all the time but 9999 ebuilds aren't bumped. Those who can't deal with the breakage shall not use live ebuilds.

**Comment by SoapGentoo** [comment]
@0xd34df00d I agree with @a17r here, `-9999` cannot be used in that way.

**Comment by 0xd34df00d** [comment]
@SoapGentoo done.

**Comment by SoapGentoo** [comment]
@0xd34df00d last request: please don't end COMMITMSGs with a full-stop, as COMMITMSGs aren't full sentences usually

**Comment by 0xd34df00d** [comment]
@SoapGentoo done!

**Comment by kensington** [comment]
Thanks, picked as f3973a9cb0583bea0b901ec2f3dc17960b349817...618d97a850aa4b933f47817cc3ed109fec928808.

**Comment by SoapGentoo** [comment]
@0xd34df00d could you please start nixing the leechcraft project from existence and modernizing the eclass to EAPI 6?

**Pull request closed without merging**